### PR TITLE
additional doc re: Tungsten

### PR DESCRIPTION
**Pull request by groenewold-pythian**
Related issue: #371 

### Description

This PR adds some additional documentation for running gh-ost in Tungsten scenarios

### CODE REVIEW

File: doc/cheatsheet.md [hunk 1]
```diff
@@ -146,8 +146,12 @@ gh-ost --allow-master-master --assume-master-host=a.specific.master.com
...
-If you choose to migrate directly on master (see above), there's nothing special you need to do. If you choose to migrate via replica, then you must supply the identity of the master, and indicate this is a tungsten setup, as follows:
+If you choose to migrate directly on master (see above), there's nothing special you need to do. 
+
+If you choose to migrate via replica, then you need to make sure Tungsten is configured with log-slave-updates parameter (note this is different from MySQL's own log-slave-updates parameter), otherwise changes will not be in the replica's binlog, causing data to be corrupted after table swap. You must also supply the identity of the master, and indicate this is a tungsten setup, as follows:
 
 ```
 gh-ost --tungsten --assume-master-host=the.topology.master.com
 ```
+
+Also note that `--switch-to-rbr` does not work for a Tungsten setup as the replication process is external, so you need to make sure `binlog_format` is set to ROW beforehand.
```
Comment (shlomi-noach) [inline doc/cheatsheet.md]: Let's also clarify what "beforehand" means. I _think_ what it should mean is that `binlog_format=ROW` before tungsten made connection to the server. Correct?

Comment (groenewold-pythian) [inline doc/cheatsheet.md]: Yes that is correct

Comment (shlomi-noach) [inline doc/cheatsheet.md]: Would you like to update this sentence?

Comment (groenewold-pythian) [inline doc/cheatsheet.md]: Done. Let me know if that looks better now. Thanks!

**Comment by shlomi-noach** [comment]
Thank you!

**Pull request merged**